covidfaq/evaluating/model/es_enhanced_reranker.py

```diff
@@ -0,0 +1,66 @@
+import logging
```

Comment:
what is the difference between this ES and the other?
I think it's better if we make this one subclass of the other and we only change the code related to the differences..

Reply:
Good idea.